.circleci/config.yml

```diff
@@ -307,10 +297,10 @@ workflows:
           requires:
             - build-theme
       - cypress:
-          filters:
-            branches:
-              only:
-                - test
+          # filters:
+            # branches:
+              # only:
+                # - test
```

Comment:
Let's leave the filter because we definitely don't want it to run on prod:
```
      - cypress:
          filters:
            branches:
              only:
                - dev
                - stage
                - test
```